site-packages/integralstor_utils/disks.py

```diff
@@ -217,6 +217,36 @@ def get_partitions(disk_name):
     else:
         return l, None
 
+def get_os_partition_stats():
+    ret_list = []
+    try:
+        lines, err = command.get_command_output('df -HT --exclude-type=devtmpfs --exclude-type=tmpfs --exclude-type=zfs')
+        if err:
+            raise Exception(err)
+        #print lines, err
+        num=0
+        ret_list = []
+        for line in lines:
+            if num == 0:
+                num += 1
+                continue
+            components = line.split()
+            tmp_dict = {}
+            #print components
+            percentage_str = components[5]
+            percentage_pos = percentage_str.find('%')
+            percentage = int(percentage_str[:percentage_pos])
+            tmp_dict['percentage_free_str'] = percentage
```

Comment:
Since `percentage` variable is `int` type, `tmp_dict['percentage_free_str']` will also be `int`. Do we intend to have it as a `string`?